From 2197a5d63ec042d7d0acf0cae2f7faf439ca1976 Mon Sep 17 00:00:00 2001
From: Zhao Chenhui <chenhui.zhao@freescale.com>
Date: Sun, 28 Apr 2013 16:10:31 +0800
Subject: [PATCH 341/547] powerpc/dts: add power node for p1023

Change-Id: Iaa01d6bd23139f73c563b2218ba28f9ef96212bb
Signed-off-by: Zhao Chenhui <chenhui.zhao@freescale.com>
Reviewed-on: http://git.am.freescale.net:8181/2545
Reviewed-by: Wang Dongsheng-B40534 <dongsheng.wang@freescale.com>
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Li Yang-R58472 <LeoLi@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/powerpc/boot/dts/fsl/p1023si-post.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/powerpc/boot/dts/fsl/p1023si-post.dtsi b/arch/powerpc/boot/dts/fsl/p1023si-post.dtsi
index 95c4ac0..334ffba 100644
--- a/arch/powerpc/boot/dts/fsl/p1023si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/p1023si-post.dtsi
@@ -292,6 +292,11 @@
 		fsl,has-rstcr;
 	};
 
+/include/ "pq3-power.dtsi"
+	power@e0070 {
+		compatible = "fsl,p1023-pmc", "fsl,mpc8548-pmc";
+	};
+
 	fman0: fman@100000 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-- 
1.8.4.93.g57e4c17

